### DISCUSSION TITLE
WoW Talent Tree Manager v1.4.0

### DISCUSSION THREAD

**Discussion by TobiasM95** [TobiasM95/WoW-Talent-Tree-Manager#23] [General]
This is another big milestone update. There is a new screen that allows for the analysis of SimC results after generating profiles for simming beforehand. Right now it's a bit clunky to use and requires running sims locally (or at least grabbing the results text file from raidbots). There is a short instruction as a hover tooltip in the analysis screen on how to use.
Even when not using the analysis screen, there are now export buttons for SimC strings (for your whole loadout or single skillsets). Additionally, TTM now supports the in-game talent string format and is able to import and export those strings correctly.
The auto-updater is iterated upon and a bit better but there's still a lot of work to be done in the future to make the experience even more seamless.
Another big feature is skillset hover previews in most of the screens where there's skillsets listed without being able to see what they are. This should massively increase quality of life while using TTM.

- SimC analysis screen version 1
- SimC export strings
- Ingame skillset string import/export
- Auto-updater improvements
- Skillset previews on hover in most screens
- Better loadout solver experience

**Full Changelog**: https://github.com/TobiasM95/WoW-Talent-Tree-Manager/compare/v1.3.8...v1.4.0

<hr /><em>This discussion was created from the release <a href='https://github.com/TobiasM95/WoW-Talent-Tree-Manager/releases/tag/v1.4.0'>WoW Talent Tree Manager v1.4.0</a>.</em>

**Comment by knehighprinter** [comment]
How do i  export an edited tree for someone to view.  or post in a forum?

**Reply by TobiasM95** [reply]
When you have the tree opened, switch to the "Talent Tree Editor" tab at the top left.
Then on the top right, select "Save/Load Trees" -> Click on "Import/Export" -> Select one of the 4 export methods. 
If you want to share an export string so that others can import it in TTM then "Export to pastebin [...]" is probably your best bet.
"Export readable talent tree" creates a wall of text that describes every talent in detail so that others don't need TTM to view it but it's a long read.

**Reply by Senari7** [reply]
Is there anything in the works to make the created trees viewable and interactable without others having TTM? I'm interested in using this program for future WoW class concepts and would like to make talent trees people can easily view, but obviously expecting everyone to have TTM is not feasible and a wall of text that describes every talent is also not appealing, haha. 

Thanks!

**Reply by TobiasM95** [reply]
The only way to achieve this would be a TTM web version which is currently not in the works. So besides screenshots, and import string and a readable version, you'd have to create something from scratch yourself I'm afraid.
A web version is not in the works in is not planned for the foreseeable future.
